src/items/tables/android_subtypes.json

```diff
@@ -0,0 +1,56 @@
+{
+  "_id": "b1QbeXZGdHFIe460",
+  "name": "Android subtypes",
+  "description": "",
+  "displayRoll": true,
+  "formula": "1d3",
+  "img": "icons/svg/mystery-man.svg",
+  "replacement": true,
+  "results": [
+    {
+      "_id": "7seudUt30ZxN1Wuv",
+      "type": 2,
+      "documentCollection": "sfrpg.races",
+      "documentId": "GHIntVN70xTt2Tlo",
+      "drawn": false,
+      "flags": {},
+      "img": "https://assets.forge-vtt.com/bazaar/core/icons/svg/mystery-man.svg",
```

Comment:
All of these Forge links need to be replaced with stock Foundry links. `icons/svg/mystery-man.svg`